packages/components/RadioGroup/src/__snapshots__/RadioButton.test.tsx.snap

```diff
@@ -26,7 +26,8 @@ exports[`RadioButton component tests RadioButton default 1`] = `
   onBlur={[Function]}
   onClick={[Function]}
   onFocus={[Function]}
-  onPress={[Function]}
+  onMouseEnter={[Function]}
+  onMouseLeave={[Function]}
   onResponderGrant={[Function]}
```

Comment:
not sure why onPress is disappearing from the snapshot. The radio buttons still seem to update for me onPress. This probably requires investigation.

Reply:
Actually.. I think it's because onPress got absorbed by Pressability, and doesn't show up as a prop on the final rendered component anymore.